embedchain/models/vector_dimensions.py

```diff
@@ -4,6 +4,7 @@
 # vector length created by embedding fn
 class VectorDimensions(Enum):
     GPT4ALL = 384
+    LANGCHAINGPT4ALL = 384

```

Comment:
Dont need this. You can use the one defined above this.

Reply:
Cool